.github/CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-*       @bethqiang @juanfriss @mrtly @sanatauqir92 @pthulen @dysnomian
+*       @bethqiang @juanfriss @mrtly @sanatauqir92 @pthulen @dysnomian @erin-doyle
```

Comment:
while we're here, can you add @shannamurry here too?

Reply:
absolutely!